scripts/core/editor3/helpers/highlights.js

```diff
@@ -246,22 +246,26 @@ export function canAddHighlight(editorState, highlightType) {
  * @param {Integer} offset
```

Comment:
This is missing docs for 2 last parameters

Reply:
Done